.github/workflows/run-full-test.yaml

```
@@ -0,0 +1,29 @@
+name: Run full tests
+on:
+  schedule:
+    - cron: '0 1 * * 6'
+  workflow_dispatch:
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: |
+        set -x
+        git fetch
+        if gh pr checkout full-tests
+        then
+          gh pr close --comment 'Recreating'
+          git checkout master
+          git branch -d full-tests
```

It might fail without `-D`

> Use `-D` instead if you want to force the branch to be deleted, even if it hasn't been pushed or merged yet.

```suggestion
          git branch -D full-tests
```